tests/bootstrap.sh

```diff
@@ -7,7 +7,7 @@ if [ "${CI:-false}" = "false" ]; then
   source docker-selenium-tests/bin/activate
 fi
 
-python -m pip install selenium==4.3.0 \
+python -m pip install selenium==4.7.2 \
```

Comment:
Shouldn't this be like that?
```suggestion
python -m pip install selenium==4.7.1 \
```

Reply:
Most recent Python Selenium release is https://pypi.org/project/selenium/4.7.2/